cuegui/cuegui/HostMonitorTree.py

```diff
@@ -69,48 +69,49 @@ def __init__(self, parent):
                            "indication that a frame is using more cores than it\n"
                            "reserved. If all cores are reserved and the percentage\n"
                            "is way below 100% then the cpu is underutilized.")
-        self.addColumn("Swap", 60, id=4,
+        self.addColumn("Swap", 70, id=4,
                        data=lambda host: cuegui.Utils.memoryToString(host.data.free_swap),
                        sort=lambda host: host.data.free_swap,
                        delegate=cuegui.ItemDelegate.HostSwapBarDelegate,
                        tip="The amount of used swap (red) vs available swap (green)")
-        self.addColumn("Memory", 60, id=5,
+        self.addColumn("Phys Memory", 70, id=5,
                        data=lambda host: cuegui.Utils.memoryToString(host.data.free_memory),
                        sort=lambda host: host.data.free_memory,
-                       tip="The amount of used memory (red) vs available gpu memory (green)")
-        self.addColumn("GPU Memory", 60, id=6,
+                       delegate=cuegui.ItemDelegate.HostMemBarDelegate,
+                       tip="The amount of used memory (red) vs available phys memory (green)")
+        self.addColumn("GPU Memory", 70, id=6,
                        data=lambda host: cuegui.Utils.memoryToString(host.data.free_gpu_memory),
                        sort=lambda host: host.data.free_gpu_memory,
                        delegate=cuegui.ItemDelegate.HostGpuBarDelegate,
                        tip="The amount of used gpu memory (red) vs available gpu memory (green)")
-        self.addColumn("freeMcp", 60, id=7,
+        self.addColumn("Total Memory", 60, id=7,
+                       data=lambda host: cuegui.Utils.memoryToString(host.data.memory),
+                       sort=lambda host: host.data.total_memory,
+                       tip="The total amount of available memory.\n\n"
+                           "Takes into consideration free memory and cached memory.")
+        self.addColumn("Idle Memory", 60, id=8,
+                       data=lambda host: cuegui.Utils.memoryToString(host.data.idle_memory),
+                       sort=lambda host: host.data.idle_memory,
+                       tip="The amount of unreserved memory.")
+        self.addColumn("Tmp available", 70, id=9,
```

Comment:
I suggest a change to `Temp Dir free space`

Reply:
Too large for a column name. I'm changing it to `Temp available` instead of `Tmp available`

```suggestion
        self.addColumn("Temp available", 70, id=9,
```